### ISSUE TITLE
Poloniex FetchBalance ExchangeNotAvailable not found

### ISSUE THREAD

**Issue by Juanperezc** [ccxt/ccxt#891]
PASTE_YOUR_TEXT_DESCRIPTION_HERE

- OS: Windows 10
- Language: php
- CCXT version: 1.10
- Exchange: poloniex
- Method: fetchBalance

Hi, I am trying to get the error of api key invalid with a try in poloniex but it throws me this error.
in bittrex if it works
```
try{
  $id = "\\ccxt\\poloniex";
  $exchange = new $id();
  $exchange->apiKey = 'key';
  $exchange->secret = 'secret';
  $balance = $exchange->fetch_balance()["BTC"];
  //var_dump ($exchange->loadMarkets());
  
  //var_dump($exchange->mar)
  var_dump ($balance); // one day
} catch (\ccxt\RequestTimeout $e) {
      echo '[Timeout Error] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (\ccxt\DDoSProtection $e) {
      echo '[DDoS Protection Error] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (\ccxt\AuthenticationError $e) {
      echo '[Authentication Error] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (\ccxt\ExchangeNotAvailable $e) {
      echo '[Exchange Not Available] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (\ccxt\NotSupported $e) {
      echo '[Not Supported] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (\ccxt\NetworkError $e) {
      echo '[Network Error] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (\ccxt\ExchangeError $e) {
      echo '[Exchange Error] ' . $e->getMessage () . ' (ignoring)' . "\n";
  } catch (Exception $e) {
      echo '[Error] ' . $e->getMessage () . "\n";
  }
  
COPYPASTE_YOUR_CODE_TO_REPRODUCE_THE_BUG_HERE
DO_NOT_DELETE_THE_BACKTICKS_AROUND_THE_CODE
```
```
PHP Fatal error:  Uncaught Error: Class 'ExchangeNotAvailable' not found in C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\base\Exchange.php:655
Stack trace:
#0 C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\base\Exchange.php(878): ccxt\Exchange->raise_error('ExchangeNotAvai...', 'https://polonie...', 'POST', 403, 'not accessible ...', '(possible reaso...')
#1 C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\base\Exchange.php(676): ccxt\Exchange->fetch('https://polonie...', 'POST', Array, 'command=returnC...')
#2 C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\poloniex.php(741): ccxt\Exchange->fetch2('returnCompleteB...', 'private', 'POST', Array, NULL, NULL)
#3 [internal function]: ccxt\poloniex->request('returnCompleteB...', 'private', 'POST', Array)
#4 C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\base\Exchange.php(620): call_user_func(Array, 'returnCompleteB...', 'private', 'POST', Array)
#5 [internal function]: ccxt\Exchange->ccxt\{closure}(Array)
#6 C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\base\Exchange.php(1425): call_user_func in C:\xampp2\htdocs\apibot\vendor\ccxt\ccxt\php\base\Exchange.php on line 655
```

**Comment by kroitor** [comment]
@Juanperezc can you please tell the exact version of ccxt?

**Comment by Juanperezc** [comment]
is the last version, before doing the tests update

in the composer this appears
1.10

**Comment by kornrunner** [comment]
Fix for that is available within #876, specifically within 09e889d049314bd5f5f045856703c84ea6631220

**Comment by kroitor** [comment]
@kornrunner thanks a lot, I was about to ask you this exact question ) Really helps! Merging it now!

**Comment by Juanperezc** [comment]
Thanks @kornrunner How do I lower that commit you made? I'm sorry, I'm new to this git

**Comment by kornrunner** [comment]
I think this bug first appeared in 1.10.392, so you could try version 1.10.391, or await for @kroitor to merge my PR and issue a new release.

**Comment by kroitor** [comment]
@Juanperezc you can either clone it directly with git, or just hold on until I upload a new version with a fix. Thanks to @kornrunner and other contributors, this won't be a long wait, really ) I will let you know when it's there.

**Comment by Juanperezc** [comment]
It's fine, I'll wait better. to have it updated without these errors

**Comment by kroitor** [comment]
This is fixed in #876, ccxt version 1.10.412. Let us know if you have any difficulties with it. Thx!
